pandas/conftest.py

```diff
@@ -367,28 +367,39 @@ def unique_nulls_fixture(request):
 TIMEZONES = [None, 'UTC', 'US/Eastern', 'Asia/Tokyo', 'dateutil/US/Pacific',
              'dateutil/Asia/Singapore', tzutc(), tzlocal(), FixedOffset(300),
              FixedOffset(0), FixedOffset(-300)]
+TIMEZONE_IDS = ['None', 'UTC', 'US/Eastern', 'Asia/Tokyp',
```

Comment:
a lot of the edits here look orthogonal to the rest of the PR.  is that accurate?

Reply:
The changes in conftest are introduced already by #23982 (upon which this PR builds), because the debugging of the tz-tests is unnecessarily hard without knowing for which tz it failed.

Reply:
@h-vetinari so best to ping on that PR first.